onnxscript/optimizer/_constant_folding.py

```diff
@@ -696,6 +703,14 @@ def process_node(self, node: ir.Node):
         if any(x is None for x in input_values):
             return None
 
+        if any(input.size > self._input_size_limit for input in input_values):
```

Comment:
Looks like there might be some None cases to handle?

Reply:
Already filtered out in the line before.